test/help.test.js

```diff
@@ -45,6 +46,26 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  describe('prints right help message for each generator', function() {
+    var cmdNames = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    cmdNames.forEach(function(name) {
+      it('prints right help message for generator ' + name, function() {
+        process.env.SLC_COMMAND = 'slc loopback --help';
+        var gen = givenGenerator(name, ['--help']);
+        var helpText = gen.help();
+        var helpFileName = 'loopback_' + name + '_help.txt';
+        var helpFilePath = '../fixtures/' + helpFileName;
+        var output = fs.readFileSync(helpFilePath, 'utf8');
+        assert.equal(output, helpText);
+      });
+    });
+    afterEach(function() {
```

Comment:
Nitpick: add an empty line before `afterEach` to serve as a visual delimiter. Also please use a named function, see http://loopback.io/doc/en/contrib/style-guide.html#layout-of-test-files